SpatialGDK/Extras/schema/tombstone.schema

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
To get this done as quickly as possible, i'm not using Dynamic Components for the first pass, but we can do that in a subsequent revision.

Reply:
I've tested in NWX's r02 level and verified that this fixes the problems with harvested ore reappearing for new clients, or clients that were on the other side of the level when the harvesting took place.